### ISSUE TITLE
Betrag der Barzahlung getrennt nach Steuersätzen

### ISSUE THREAD

**Issue by andreasschiestl** [a-sit-plus/at-registrierkassen-mustercode#445]
Hallo!

Für meine Frage nehme ich mal das Beispiel aus dem Erlass unter Punkt 4.6.6.

Ein Kunde kauft in einem Geschäftsvorfall folgende Produkte:
```
100 Euro Getränke (20% USt, darin enthalten 16,67 Euro USt)
100 Euro Speisen (10% USt, darin enthalten 9,09 Euro USt)
100 Euro Wertgutschein (steuerfrei)
300 Euro Summe
```

nun muss sich auf der Rechnung eine Aufschlüsselung der USt-Beträge wiederfinden, in diesem Beispiel:
```
20% USt von 83,33 = 16,67
10% USt von 90,91 = 9,09
0% USt von 100,00 = 0,00
```

soweit sogut, nun zahlt der Kunde wie im Beispiel des Erlasses nur 100€ bar und der Restbetrag bleibt offen. Der Betrag wird nun aliquot aufgeteilt:
```
20% USt = 33,33
10% USt = 33,33
0% USt = 33,34
```

In §11 (1) RKSV steht nun:
> 3. Betrag der Barzahlung getrennt nach Steuersätzen

Heißt das nun tatsächlich dass auch diese zweite Aufschlüsselung am Beleg gedruckt werden muss?

**Comment by ErichFreitag** [comment]
Wenn ich es richtig verstanden habe ist am Beleg NUR die zweite Aufschlüsselung anzugeben, nämlich das, was den tatsächlichen Barumsatz betrifft. Die Rechnung selbst hat ja damit zunächst nichts zu tun.

**Comment by andreasschiestl** [comment]
Ok, aber nehmen wir nun an, dass der Beleg gleichzeitig die Rechnung ist (wie zB im Handel üblich), man wird hier dann ja kaum die Rechnung und einen extra Zahlungsbeleg bekommen (habe bisher keine Beleg mit QR-Code in "freier Wildbahn" gesehen)

**Comment by AxelKutschera** [comment]
Wir unterstützen schon viele RegK mit QR-Code - die erste läuft seit 1.8.2016 mit QR-Code. :-)

Anmerkung:
Ein Beleg und eine Rechnung ist etwas anderes.
Eine Urkunde kann ein Beleg nach RKSV sien und die Rechungsmerkmale enthalten, wird somit gleichzeitig eine Rechnung.

Aber nun zur Frage - Wir empfehlen es für fiskaltrust so:

1. Es werden die Rechnungsmerkmalen wie beschrieben gedruckt.
2. Die nach den Steuersätzen getrennten Beträge werden zB auch mit der KA-ID ganz klein zum QR-Code gedruckt.

_33,33_33,33_0,00_33,33_0,00_

Ich habe absichtlich alle Beträge abgerundet. Der Summenzähler sollte dennoch +100,00 sein.
Sonst stimmt es bei den einzelnen Beträgen doch nicht. Und: Ihre Datenbank wird vermutlich auch so rechnen?

**Comment by lebail** [comment]
Ja, eine Urkunde kann ein Beleg nach RKSV sein und die Rechungsmerkmale enthalten, wird somit gleichzeitig eine Rechnung. Und ich denke, dass sich die Frage auf genau diesen Fall bezog. Denn in vielen Branchen ist es üblich, Barzahlungsbelege ("Kassenbons") auszustellen, die gleichzeitig Rechnungen gemäß UStG sind.

In diesem Fall hat man eventuell ein Problem bei der Gestaltung der Urkunde, wenn der Kunde vorerst nur einen Teil des Rechnungsbetrages bezahlt.

Ich sehe als Lösungsmöglichkeit z.B., dass man die ursprüngliche Urkunde wie hier beschrieben ausstellt (Rechnung über den Gesamtbetrag mit integriertem RKSV-Barzahlungsbeleg über den tatsächlich bezahlten Betrag). Wenn dann der Rest bezahlt wird, stellt man eine (deutlich als Duplikat hinsichtlich UStG gekennzeichnete) Rechnungskopie aus (wieder über den Gesamtbetrag), der enthaltene Barzahlungsbeleg enthält diesmal aber den jetzt bezahlten Restbetrag. Die Rechnungskopie muss natürlich dieselbe Rechnungsnummer wie die ursprüngliche Rechnung haben, der enthaltene Barzahlungsbeleg muss aber eine neue RKSV-Belegnummer enthalten. Das geht nur, wenn man Rechnungsnummern und RKSV-Belegnummern unabhängig voneinander führt (was zulässig ist).

Zu den Belegnummern siehe auch https://github.com/a-sit-plus/at-registrierkassen-mustercode/issues/110

Ich mache es derzeit (für mein eigenes Unternehmen) anders - und würde mich auch über Meinungen zu dieser Variante freuen:

Um die USt-Aufteilerei, damit verbundene Rundungsdifferenzen und die notwendige Ausstellung von Rechnungsduplikaten zu vermeiden, bekommt meine Rechnung im Fall der Zahlung nur eines Teilbetrages einen zusätzlichen Posten mit negativem Betrag und dem Titel "Übertrag auf nächste Rechnung", in obigem Beispiel wären das -200 €. Dadurch wird der Rechnungsbetrag gleich dem Barzahlungsbetrag, und man kann problemlos einen RKSV-Barbeleg auf das Dokument drucken.

Anlässlich der Zahlung des Restbetrages wird eine neue Rechnung ausgestellt. Auf der steht dann "Übertrag von Rechnung Nr. 2017-XXX" und in obigem Beispiel ein Betrag von 200 €. Wieder kann man einen ganz normalen Barzahlungsbeleg aufdrucken. Dieses Verfahren funktioniert auch, wenn man für Rechnungs- und Belegnummer dieselbe Nummer benutzt, da bei der zweiten Zahlung ja eine neue Rechnung augestellt wird.

Der Übertrag enthält bei mir auf beiden Rechnungen 0% USt. (Argument: ich habe dem Kunden einfach den entsprechenden Betrag geliehen - unabhängig von der erbrachten Leistung). Das führt natürlich dazu, dass bereits auf dem Barbeleg des _ersten_ Dokumentes die _gesamte_ USt. ausgewiesen ist, obwohl der Kunde zu diesem Zeitpunkt nur einen Teil davon bezahlt hat. Dies kann in manchen Fällen (Ist-Versteuerer) bewirken, dass der Unternehmer die USt. bereits früher abführen muss, als es notwendig wäre. Dafür ist das Ganze in meinen Augen übersichtlicher (eine Rechnung = ein Barbeleg) und auch einfacher zu implementieren.

Achtung: Ich bin kein Steuerberater und weiß nicht, ob diese Variante allen Vorschriften entspricht. Ich gehe allerdings (zumindest in meinem eigenen Fall) davon aus, dass ich sie einem Prüfer schlüssig darlegen könnte. Und dass ich dadurch eventuell USt-Beträge zu früh an das Finanzamt überweise, wird in selbigem wohl niemenden stören.

**Comment by Alex1130** [comment]
Kleinbetragsrechnungen: Für Rechnungen unter 400€ kann die Aufschlüsselung auch entfallen
[USP Kleinbetragsrechnung](https://www.usp.gv.at/Portal.Node/usp/public/content/steuern_und_finanzen/rechnung/kleinbetragsrechnungen/40814.html)

**Comment by lebail** [comment]
@Alex1130: Auf der _Rechnung_ schon. Aber nicht auf dem _Barzahlungsbeleg_, denn dort ist sie von der RKSV ausnahmslos vorgeschrieben.

**Comment by Alex1130** [comment]
@lebail IMHO reicht eine Brutto Aufschlüsselung
 dazu [BAO](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10003940&FassungVom=2017-04-01)
§ 132a
...
(3) Die Belege haben mindestens folgende Angaben zu enthalten:						
1. eine eindeutige Bezeichnung des liefernden oder leistenden Unternehmers oder desjenigen, der gemäß Abs. 2 an Stelle des Unternehmers einen Beleg erteilen kann,
2. eine fortlaufende Nummer mit einer oder mehreren Zahlenreihen, die zur Identifizierung des Geschäftsvorfalles einmalig vergeben wird,
3. den Tag der Belegausstellung,
4. die Menge und die handelsübliche Bezeichnung der gelieferten Gegenstände oder die Art und den Umfang der sonstigen Leistungen und
**5. den Betrag der Barzahlung, wobei es genügt, dass dieser Betrag auf Grund der Belegangaben rechnerisch ermittelbar ist.**
...
 und [RKSV](https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=20009390&FassungVom=2017-04-01)
§ 11. (1) Auf dem Beleg sind neben den Belegdaten des § 132a Abs. 3 BAO folgende Daten auszuweisen:										
1. Kassenidentifikationsnummer
2. Datum und Uhrzeit der Belegausstellung
**3. Betrag der Barzahlung getrennt nach Steuersätzen**
4. Inhalt des maschinenlesbaren Code.
...

**Comment by andreasschiestl** [comment]
@AxelKutschera Danke für die Einschätzung. Ich bin persönlich nicht so der Fan von (für den Normalanwender und Kunden) kryptischen Bezeichnungen, habe schon Kassen gesehen welche die Kassen-ID ohne irgendeine Beschriftung drücken und dann noch dazu unerklärliche Zufallsnummer. Da steht dann am Beleg "6592163" und niemand weiß was diese Nummer ist.

Aber wir werden die RKSV-Infos wie Kassen-ID, RKSV-Belegnummer (hat einen eigenen Nummernkreis) usw. zum QR-Code drucken, und das irgendwie so gestalten dass man einen Zusammenhang erkennt :)

Zu Ihrer Frage bzgl der Rundung von 33,33 auf 33,34 für einen Steuersatz. Das wird exakt so gerechnet und in der Datenbank gespeichert und ist steht auch im Erlass so:

> Nach der
RKSV sind die 100 Euro aliquot aufzuteilen und somit je Steuersatz (20%, 10% und 0%) mit
je 33,33 Euro im DEP festzuhalten. Die Rundungsdifferenz in Höhe von einem Cent ist einem
Steuersatz zuzuschlagen. Der Umsatzzähler ist um 100 Euro zu erhöhen.

Ansonsten würde die Summe der ganzen Teilbeträge in den Belegen einen anderen Summenzähler ergeben und (vermutlich, nicht getestet) bei der DEP Prüfung einen Fehler anzeigen.

**Comment by lebail** [comment]
@Alex1130: Ja, eine Aufschlüsselung der Bruttobeträge reicht natürlich. Ich dachte, es ging darum, ob überhaupt eine Aufschlüsselung gemacht werden muss.

**Comment by Flanelli** [comment]
Rechnungen ( also alles was Ziel oder Teilzahlung hat ) und Zahlungen trennen
dann gibt es kein Kopfweh mehr.

Kunde kauft Ware um 230 Euro aber bezahlt erst irgendwann
oder bezahlt nur einen Teil.

Rechnung wird erstellt ohne Signierung da Ziel, oder Teilzahlung
Belegnummer Rechnung ist eigener Kreis
Andruck der Aufschlüsselung
Ust20% Netto 100,00 Ust 20,00 Brutto 120,00
Ust10% Netto 100,00 Ust 10,00 Brutto 110,00

Kunde bezahlt Teilbetrag, Restbetrag oder Alles
Barverkaufsbeleg ( Kassabon ) wird erstellt und signiert
Bezug auf die Rechnungsnummer wird angedruckt.
Falls noch etwas offen bleibt wird der Betrag als Hinweis angedruckt
Es erfolgt kein Andruck einer Aufschlüsselung nach Ustsätzen
da solche Zahlungen generell IMMER auf Betragsatz-Null-Ust laufent.

Somit lediglich Erstellung einer Rechnung
und je nach Situation der Zahlung(en) der/die
Zahlungsbeleg(e)

**Comment by andreasschiestl** [comment]
Ja Rechnung und Zahlungsbeleg zu trennen ist eine Möglichkeit. aber leider nicht immer erwünscht. Einfach aus dem Grund, weil mehr gedruckt werden muss.

**Comment by Flanelli** [comment]
@andreasschiestl 
Das mit dem Mehrdruck trifft aber nur dann zu wenn die Rechnung akontiert wird.
Erfolht die Bezahlung gleich zur Gänze dann wird ja ein normaler Kassenbeleg gedruckt der
ja nun mit der RKSV ohnehin eigentlich alle Merkmale einer Rechnung aufweist.

Ist es nur ein Aconto dann müssen sie ja sowieso bei jedem weiteren Aconto
und auch bei der Restzahlung  einen weiteren Zahlungsbeleg drucken.
Somit hält sich wohl die << Papierverschwenduing >> in engen Grenzen.

**Comment by daiNc** [comment]
Hey, 
tut mir leid aber könnte mir jemand weiterhelfen? Ich finde die Beschreibung der Teilzahlung einer Rechnung und die Aufteilung der Steuersätze nirgends beschrieben.

> Nach der
RKSV sind die 100 Euro aliquot aufzuteilen und somit je Steuersatz (20%, 10% und 0%) mit
je 33,33 Euro im DEP festzuhalten. Die Rundungsdifferenz in Höhe von einem Cent ist einem
Steuersatz zuzuschlagen. Der Umsatzzähler ist um 100 Euro zu erhöhen.

**Comment by ErichFreitag** [comment]
Erlass vom 04.08.2016 Punkt 4.6.6
